Library/TestHelpers/TestCase.swift

```diff
@@ -48,7 +49,7 @@ internal class TestCase: FBSnapshotTestCase {
       debounceInterval: .seconds(0),
       device: MockDevice(),
       facebookAppDelegate: self.facebookAppDelegate,
-      isVoiceOverRunning: const(false),
```

Comment:
What did the compiler not like about this?

Reply:
If I use `const(false)`, the tests will fail with the message `Extraneous argument label 'isVoiceOverRunning:' in call`.